### PR TITLE
integration/kubernetes: Strength the k8s-block-volume test

### LINKED ISSUE #3123: k8s-block-volume test seems to fail intermittently
Looking at the recent failed jobs on:

http://jenkins.katacontainers.io/job/kata-containers-2.0-fedora-PR/

I noticed the jobs 615, 612, 598 failed because of integration/kubernetes/k8s-block-volume.bats. On other jobs the test passed. It might indicate a intermittent failure.

### PR DESCRIPTION
**Pull request by wainersm**
Wait on the created volume to be "Available" before proceed with
the remaining of the test; Also refactor the code to use variables
that hold the name of the resources.

Fixes #3123

Signed-off-by: Wainer dos Santos Moschetta <wainersm@redhat.com>

### CODE REVIEW

**Review by amshinde — APPROVED** [review]

**Comment by amshinde** [comment]
Thanks @wainersm, lets hope this solves the block-volume test.

File: integration/kubernetes/k8s-block-volume.bats [hunk 1]
```diff
@@ -33,6 +37,8 @@ setup() {
 	sed -i "s|HOSTNAME|$(hostname)|" "$tmp_pv_yaml"
 	sed -i "s|CAPACITY|${vol_capacity}|" "$tmp_pv_yaml"
 	kubectl create -f "$tmp_pv_yaml"
+	cmd="kubectl get pv/${volume_name} | grep Available"
+	waitForProcess "$wait_time" "$sleep_time" "$cmd"
```
Comment (amshinde) [inline integration/kubernetes/k8s-block-volume.bats:41]: I was hoping that we could make use of `kubectl wait` here, but looks like it does not work for persistent volume resources.

Comment (wainersm) [reply integration/kubernetes/k8s-block-volume.bats:41]: @amshinde same here. I tried to use `kubectl wait` but found no way to check the status is available. :(

Comment (egernst) [reply integration/kubernetes/k8s-block-volume.bats:41]: same concern. This is at least better than what we had...

**Review by egernst — APPROVED** [review]

**Pull request merged**